internal/driver/commands.go

```diff
@@ -284,7 +287,16 @@ func usage(commandLine bool) string {
 			radioOptions[vr.group] = append(radioOptions[vr.group], name)
 			continue
 		}
-		variables = append(variables, fmtHelp(prefix+name, vr.help))
+		option := prefix + name
+		if vr.kind == intKind {
```

Comment:
Should there be something like `option=<s>` for stringKind?

Reply:
Yeah, I was debating whether that's necessary. Usage for options like "disasm" is actually quite complicated: 

```
(pprof) help disasm
Output assembly listings annotated with samples
  Usage:
    disasm<func_regex|address> [-focus_regex]* [-ignore_regex]* >f
    Include functions matching func_regex, or including the address specified.
    Include samples matching focus_regex, and exclude ignore_regex.
    Optionally save the report on the file f
```

I am not sure `option=<s>` is actually a good indication of the usage or we should just encourage people to use `help` to understand each individual command.

Reply:
Per offline discussion, added `option=<s>` for every string kind option.